index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Diego Vidal</title>
+    <link rel="stylesheet" href="style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <!-- <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap" rel="stylesheet"> -->
+
+</head>
+
+<body>
+    <div class="body-wrap">
+
+        <div class="nav-flex-container">
+            <div class="nav-logo">My Logo</div>
+            <button class="nav-menu-button"><img class=" nav-logo" src="./icons/Icon - Menunav-button.png"
+                    alt="navegation button"></button>
+            <p>Portfolio</p>
+            <p>About</p>
+            <p>Contact</p>
+        </div>
```

Comment:
Using the header tag to wrap the navigation, makes it very semantic and accessible, so i suggest wrapping the nav with an `h1` tag.

Furthermore, i encourage you to reduce the left and right spaces on the navigation in order to make it match the template.

Your design
<img width="218" alt="nav" src="https://user-images.githubusercontent.com/84930355/235004967-8d828aa5-b83d-4c59-a2c7-37ec7cc0e8d6.PNG">


Figma
<img width="283" alt="navvv" src="https://user-images.githubusercontent.com/84930355/235005312-6895ca1d-08f3-43e6-8102-7f0e8ab0f240.PNG">

Reply:
So, I should move the current h1(including the nested text) in order be wrapping the nav bar? Or I should change the current h1 to h2, and use the tag h1 to wrap the nav bar? would be great if you could give me more insight here? Thanks for all your feedback!